src/MessagePack/Internal/ReflectionExtensions.cs

```diff
@@ -30,14 +35,24 @@ public static bool IsAnonymous(this System.Reflection.TypeInfo type)
                    && type.IsDefined(typeof(CompilerGeneratedAttribute), false);
         }
 
+        public static bool IsAnonymous(this Type type)
```

Comment:
Since the containing type is `internal`, just change the original method signature to take `Type` rather than duplicate it to another overload, please.